dvc/repo/experiments/queue/remove.py

```diff
@@ -58,7 +58,8 @@ def remove_tasks(  # noqa: C901
             result: AsyncResult = AsyncResult(task_id)
             if result is not None:
                 result.forget()
-            celery_queue.celery.purge(msg.delivery_tag)
+            if msg.delivery_tag:
+                celery_queue.celery.purge(msg.delivery_tag)
```

Comment:
@karajan1001 This changes look unrelated

Reply:
Yes, there might be something wrong with My environment, without these changes I can't pass `pre-commit` check, I will create a separate PR for them